examples/website/src/shared/query.tsx

```diff
@@ -1,6 +1,6 @@
 import gql from 'graphql-tag'
 import {articleMetaDataFragment, simpleImageDataFragment} from './route/gqlFragments'
-import {QueryHookOptions, useQuery} from 'react-apollo'
+import {QueryHookOptions, useQuery} from '@apollo/client'
```

Comment:
Are there any side effects of that change? :eyes:

Reply:
Maybe. But i didn't see any in the editor.